u2-routing-and-controllers/features/userController.js

```diff
@@ -0,0 +1,45 @@
+const { findUser, findUserByUsername, findUserBySearch } = require('../_services/fakedb')
+
+const getUserByQuery = (req, res) => {
+  const UserId = req.query.id
```

Comment:
ฉันแนะนำให้คุณใช้ `lowerCamelCase` สำหรับชื่อตัวแปร: `const userId = ...`
I encourage you to use `lowerCamelCase` for variable names: `const userId = ...`